docker-compose.yml

```diff
@@ -0,0 +1,13 @@
+version: "2.2"
```

Comment:
* Update the readme with steps on how/when/why to run this
* Can you make sure it's the version of PG that's the same as the managed DB?